metadata/src/main/java/com/redhat/lightblue/metadata/mongo/MongoMetadata.java

```diff
@@ -137,12 +102,10 @@ public EntityMetadata getEntityMetadata(String entityName,
             }
             return new EntityMetadata(info, schema);
         } catch (Error | IllegalArgumentException e) {
-            // rethrow lightblue error or illegao arg exception
+            // rethrow lightblue error or IllegalArgumentException
             throw e;
         } catch (Exception e) {
-            // throw new Error (preserves current error context)
-            LOGGER.error(e.getMessage(), e);
```

Comment:
Because analyzeException method dos already log that (see first line of analyzeException method https://github.com/luan-cestari/lightblue-mongo/blob/errorCodeTimeoutrest%2312/metadata/src/main/java/com/redhat/lightblue/metadata/mongo/MongoMetadata.java#L663 )